container-storage-setup.1

```diff
@@ -6,15 +6,19 @@ container\-storage\-setup - Tool for setting up storage for container runtimes.
 .PP
 \f[B]container-storage-setup\f[] [OPTIONS]
 
-\f[B]container-storage-setup\f[] [OPTIONS] INPUTFILE OUTPUTFILE
+\f[B]container-storage-setup\f[] [OPTIONS] COMMAND [args]
 
 .SH DESCRIPTION
-  container-storage-setup reads INPUTFILE to create storage and generate
-  OUTPUTFILE for use by container runtimes.
+  container-storage-setup sets up storage for use by container
+  runtimes.
 
-  Note: executing container-storage-setup without options will default to
-  using docker config files /etc/sysconfig/docker-storage-setup for
-  input and /etc/sysconfig/docker-storage for output.
+  First form of the command defaults to using docker config files
+  /etc/sysconfig/docker-storage-setup for input and
+  /etc/sysconfig/docker-storage for output.
+
+  In the second form, it creates and manages storage configurations.
+  Currently supported commands are create, remove, list, activate,
```

Comment:
Commands listed below.  Don't maintain the list here, otherwise we will end up with bugs.